examples/hello-world/styles.css

```
@@ -1,15 +1,119 @@
+* {
+  box-sizing: border-box;
+  font-family: 'Inter', -apple-system, sans-serif;
+}
+
+html, body {
   margin: 0;
-  background: var(--bg);
-  color: var(--text);
+  padding: 0;
+  background-color: #0f172a; /* Fallback */
+  background: linear-gradient(135deg, #0f172a 0%, #1e293b 100%);
```

Stylelint (17.9.0)

[error] Overridden property "background-color" by shorthand "background" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)